hashintel/hash
# Move `external-services` folder to `packages/hash`

## 🌟 What is the purpose of this PR?

This PR slightly cleans up monorepo root directory by moving `external-services` to `packages/hash`.




## 🔗 Related links





- [Asana task](https://app.asana.com/0/1202250835094705/1202473225833769/f) _(internal)_
- [Slack thread](https://hashintel.slack.com/archives/C022217GAHF/p1655416921012649) _(internal)_

## 📜 Does this require a change to the docs?



- No, because `package.json` scripts abstract away the location of external services and also because `var` dir is still at the top level (changing this is out of scope)

## ⚠️ Known issues

- Semgrep [revealed](https://github.com/hashintel/hash/pull/687#issuecomment-1163452061) a few issues in `Dockerfile`s. It'd be good to address them in a separate PR. 


## 🐾 Next steps

- Continue refactoring the structure of the monorepo. In particular:
   - remove `yarn dev` / `yarn external-services` from top-level package.json
   - move local `var` folder to `packages/hash/var`


## 🛡 What tests cover this?

1.  CI


## ❓ How to test this?



1.  Checkout the branch / view the deployment
1.  Launch external services via `yarn external-services up` (as before)
1.  Run the app and confirm that it works
